packages/@react-aria/interactions/src/useScroll.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {RefObject, useCallback, useEffect} from 'react';
+import {ScrollEvents} from '@react-types/shared';
+
+export interface ScrollProps extends ScrollEvents {
+  /** Whether the scroll listener should be disabled. */
+  isDisabled?: boolean
+}
+
+// scroll wheel needs to be added not passively so it's cancelable, small helper hook to remember that
+export function useScroll(props: ScrollProps, ref: RefObject<HTMLElement>): void {
```

Comment:
IMO this should be `useScrollWheel` since it specifically responds to wheel events and not e.g. scrolling via dragging the scrollbar.